src/ui/components/SearchInput/index.js

```diff
@@ -30,32 +30,34 @@ export default class SearchInput extends React.Component {
 
   onBlur = () => {
     this.setState({ focus: false });
+    if (!this.value) {
+      // Animation will start if there is no value.
```

Comment:
There is no `transitionstart` event so we need to manually detect when we will start animating.